Source/m_menu.c

```diff
@@ -5198,10 +5199,11 @@ boolean M_Responder (event_t* ev)
 
         if (M_InputActivated(input_demo_fforward))
         {
-          if (demoplayback && singledemo && !demo_skipping)
+          if (demoplayback && singledemo && !demo_skipping && !fastdemo)
           {
-            fastdemo = !fastdemo;
-            I_SetFastdemoTimer();
+            static boolean fastdemo_timer = false;
```

Comment:
SO , it's not possible to turn a fastdemo into a normal demo anymore?

Reply:
The `fastdemo` variable indicates the -fastdemo command line parameter. It is possible to enable the fastdemo timer during normal demo playback, but not with the -fastdemo parameter as it counts FPS.

Reply:
Alright! 👍